optuna/samplers/_cmaes.py

```diff
@@ -352,38 +358,36 @@ def _restore_optimizer(
 
     def _init_optimizer(
         self,
-        search_space: Dict[str, BaseDistribution],
-        ordered_keys: List[str],
+        trans: _SearchSpaceTransform,
         population_size: Optional[int] = None,
         randomize_start_point: bool = False,
     ) -> CMA:
+
+        lower_bounds = trans.bounds[:, 0]
+        upper_bounds = trans.bounds[:, 1]
+        n_dimension = len(trans.bounds)
+
         if randomize_start_point:
             # `_initialize_x0_randomly ` returns internal representations.
```

Comment:
I notice that we can remove this comment because `_initialize_x0_randomly` is removed. L374 is also.